integration-tests/src/test/java/io/druid/tests/indexer/AbstractIndexerTest.java

```diff
@@ -74,10 +72,7 @@ public Boolean call() throws Exception
 
   protected String getTaskAsString(String file) throws IOException
   {
-    InputStream inputStream = ITRealtimeIndexTaskTest.class.getResourceAsStream(file);
-    StringWriter writer = new StringWriter();
-    IOUtils.copy(inputStream, writer, "UTF-8");
-    return writer.toString();
+    return IOUtils.toString(ITRealtimeIndexTaskTest.class.getResourceAsStream(file), "UTF-8");
```

Comment:
You are right! This should be always closed.
From JavaDoc of IOUtils,
"Wherever possible, the methods in this class do not flush or close the stream. This is to avoid making non-portable assumptions about the streams' origin and further use. Thus the caller is still responsible for closing streams after use."